### PR TITLE
Update optimizations page

### PR DESCRIPTION
**Pull request by fewtarius**
Modifications to reflect new features, to include benchmarks, and updated recommendations.

### CODE REVIEW

File: docs/configure/optimizations.md [hunk 1]
```diff
@@ -10,15 +10,32 @@ Optimizing for performance will have significant impact on battery life, however
...
+|AYANEO Air Plus|6323|11394|22519|39682|43529|**47904**|47562|
+|Atari VCS|5015|6172|9027|**9726**|9239|9233|9257|
 
 #### ARM based devices
 
-|Enabled CPU Threads|Cooling Profile|Scaling Governor|Enhanced Power Saving|WIFI Power Saving|
-|----|----|----|----|----|
-|All|Moderate<sup>1</sup>|Balanced|Off|Off|
+|Enabled CPU Threads|CPU Scheduler|GPU Performance Profile|Enhanced Power Saving|Cooling Profile|WIFI Power Saving|
+|All|Performance|Best Performance|Off|Moderate<sup>1</sup> or Auto|Off|
```
Comment (anthonycaccese) [inline docs/configure/optimizations.md:38]: Add:
```
|----|----|----|----|----|----|
```

File: docs/configure/optimizations.md [hunk 2]
```diff
@@ -29,43 +46,54 @@ JELOS includes an `Enhanced Power Saving` mode which is available in the `System
...
 ### Recommended Settings
 
 Enable Enhanced Power Saving, and enable all options.  If the device has undesired behavior, disable the options that may effect stability and reboot the device.
 
 #### AMD / Intel based devices
-|Enabled CPU Threads|Cooling Profile|Max TDP|Scaling Governor|GPU Performance Profile|Enhanced Power Saving|WIFI Power Saving|
-|----|----|----|----|----|----|----|
-|4|Quiet<sup>1</sup>|4.5w|Powersave|Battery Focus|On|On|
+|Enabled CPU Threads|CPU Scheduler|Energy Performance Preference|GPU Performance Profile|Enhanced Power Saving|Cooling Profile|WIFI Power Saving|
+|4|Powersave|Power Saving|Battery Focus|On|Quiet<sup>1</sup>|On|
```
Comment (anthonycaccese) [inline docs/configure/optimizations.md:60]: Add:
```
|----|----|----|----|----|----|----|
```

**Review by anthonycaccese — COMMENTED** [review]
Added 2 updates to make to enable tables to render correctly

File: docs/configure/optimizations.md (hunk 2 above)
Comment (fewtarius) [reply docs/configure/optimizations.md:62]: Fixed up. :)

**Pull request merged**